openfl-tutorials/interactive_api/PyTorch_MVTec_PatchSVDD/envoy/shard_config.yaml

```diff
@@ -0,0 +1,6 @@
+template: mvtec_shard_descriptor.MVTecShardDescriptor
```

Comment:
All files in the envoy and workspace directories are executable, only sh-files should be executable.